Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.9
+ADD app.py .
+ADD requirements.txt .
+RUN pip install -r requirements.txt
+COPY . .
+
+ENV FLASK_APP=app.py
+
+EXPOSE 5000
+
+CMD ["flask", "run", "--host=0.0.0.0"]
```

Comment:
WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.

Il faut utiliser autre chose pour le déploiement serveur (gunicorn, waitress ou je sais plus quoi)